test/simplehashregistrar_test.js

```diff
@@ -28,6 +28,10 @@ function days(numberOfDays) {
 	return numberOfDays * 24 * 60 * 60;
 }
 
+function assertIsContractError(err) {
+	return assert.ok(err.toString().indexOf("invalid JUMP") != -1, err);
```

Comment:
This line, in conjunction with `ens/node_modules/solc/soljson.js`, seems to have caused all CI build done since to fail.